dpgen/data/arginfo.py

```diff
@@ -105,25 +105,25 @@ def init_bulk_jdata_arginfo() -> Argument:
         "init_bulk_jdata",
         dict,
         [
-            Argument("stages", list, optional=False, doc=doc_stages),
-            Argument("elements", list, optional=False, doc=doc_elements),
-            Argument("potcars", list, optional=True, doc=doc_potcars),
+            Argument("stages", list[int], optional=False, doc=doc_stages),
+            Argument("elements", list[str], optional=False, doc=doc_elements),
+            Argument("potcars", list[str], optional=True, doc=doc_potcars),
             Argument("cell_type", str, optional=True, doc=doc_cell_type),
-            Argument("super_cell", list, optional=False, doc=doc_super_cell),
+            Argument("super_cell", list[int], optional=False, doc=doc_super_cell),
```

Comment:
Is it list[int] or list[float]?

Reply:
should be `int`. see https://pymatgen.org/pymatgen.core.html#pymatgen.core.structure.Structure.make_supercell